src/components/Header.jsx

```diff
@@ -0,0 +1,31 @@
+import icon from "../assets/images/night.png";
+import style from "./Header.module.scss";
+import iconMoon from "../assets/images/moon.png";
+import { useContext } from "react";
+import { CustomContext } from "../helpers/Context";
+
+function Header() {
+  const { isTheme = true, toggleTheme } = useContext(CustomContext);
+  return (
+    <>
+      {isTheme ? (
+        <div className={style.head_block}>
+          <h1>Where in the world?</h1>
+          <h3 onClick={toggleTheme}>
```

Comment:
Why h3?

Reply:
Well, first of all it doesn't look as a heading. It's just a span to me. There is no semantical meaning for the app.
Next, we should use headings without skipping them. If there is no h2 on a page h3 is a semantic error.

Reply:
done